<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>256</width>
    <height>256</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>256</width>
    <height>256</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>256</width>
    <height>256</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>NLToolkit</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="residentBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>241</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>221</width>
       <height>54</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item row="0" column="2">
       <widget class="QPushButton" name="residentSetWallet">
        <property name="text">
         <string>Set</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="residentWalletLabel">
        <property name="text">
         <string>0 Bells</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Wallet:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>ABD:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="residentSetABD">
        <property name="text">
         <string>Set</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="residentABDLabel">
        <property name="text">
         <string>0 Bells</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>256</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>FIle</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuResident">
    <property name="title">
     <string>Resident</string>
    </property>
    <addaction name="actionResident_1"/>
    <addaction name="actionResident_2"/>
    <addaction name="actionResident_3"/>
    <addaction name="actionResident_4"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuTown">
    <property name="title">
     <string>Town</string>
    </property>
    <addaction name="actionTown_editor"/>
    <addaction name="actionTown_editor_2"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuResident"/>
   <addaction name="menuTown"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionResident_1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Resident 1</string>
   </property>
  </action>
  <action name="actionResident_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Resident 2</string>
   </property>
  </action>
  <action name="actionResident_3">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Resident 3</string>
   </property>
  </action>
  <action name="actionResident_4">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Resident 4</string>
   </property>
  </action>
  <action name="actionTown_editor">
   <property name="text">
    <string>Acre editor</string>
   </property>
  </action>
  <action name="actionTown_editor_2">
   <property name="text">
    <string>Town editor</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
